File: internal/adapters/guest/fixtures/jvm-java-spring/src/main/java/dev/mds/GreetingController.java
package dev.mds;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class GreetingController {
    @Value("${mds.greeting}")
    private String greeting;

    @GetMapping("/greeting")
    public String greeting() {
        var inspected = greeting;
        return inspected;
    }
}

File: internal/adapters/guest/fixtures/jvm-kotlin-spring/src/main/kotlin/dev/mds/GreetingController.kt
package dev.mds

import org.springframework.beans.factory.annotation.Value
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class GreetingController(@Value("\${mds.greeting}") private val greeting: String) {
    @GetMapping("/greeting")
    fun greeting(): String {
        val inspected = greeting
        return inspected
    }
}
